mocha-client-tests/karma.conf.js

```diff
@@ -0,0 +1,72 @@
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      // Test dependencies
+      'node_modules/expect.js/index.js',
+      'node_modules/sinon-chrome/bundle/sinon-chrome-webextensions.min.js',
```

Comment:
The schemas are kept up to date but the one you linked to is the schema for a Chrome compatible interface, the one documented here: https://developer.chrome.com/extensions/storage#type-StorageArea

The schema file that describes Mozilla's promise based interface is somewhere else but I can't find it at the moment! @rpl works with them a lot so maybe he can find the exact place in mozilla central.